fairlearn/reductions/grid_search_classification.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import copy
+import numpy as np
+
+from fairlearn.metrics.demographic_parity import DemographicParity
+
+
+class GridSearchClassification:
+    def __init__(self,
+                 learner,
+                 fairness_metric=DemographicParity(),
+                 number_of_lagrange_multipliers=7):
```

Comment:
This is the fallback if the user does not specify the Lagrange multipliers with the `fit` call

Reply:
this shoudl be documented

Reply:
Am cogitating on how to make the whole thing more general

Reply:
Have switched to keyword arguments on the `fit` call